pynsee/localdata/_get_geo_list_simple.py

```diff
@@ -12,9 +12,10 @@
 
 
 @lru_cache(maxsize=None)
-def _get_geo_list_simple(geo, progress_bar=False):
-
+def _get_geo_list_simple(geo, date=None, progress_bar=False):

```

Comment:
these changes need to be removed

Reply:
@tgrandje this still needs to be removed ;)

Reply:
@tfardet So I confirm this was (first removed then) restored in accordance with the latest PR on master. This should resolve the conflict.